.ci/test.py

```diff
@@ -134,6 +137,23 @@ def run_one(p: Plugin) -> bool:
         print("##[endgroup]")
 
 
+def install_pyln_testing(pip_path):
+    # Update pyln-testing to master since we're running against c-lightning master.
+    dest = Path('/tmp/lightning')
+    repo = 'https://github.com/ElementsProject/lightning.git'
+    if not dest.exists():
+        subprocess.check_call([
+            'git', 'clone', repo, '/tmp/lightning'
```

Comment:
Nit: instead of using the string `/tmp/lightning` you could use the `dest` variable you already set.